Controller/adminhtml/Faq/Delete.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\ProductFAQ\Controller\Adminhtml\Faq;
+
+use Inchoo\ProductFAQ\Api\FaqRepositoryInterface;
+use Magento\Backend\App\Action;
+
+class Delete extends Action
+{
+    /**
+     * Delete constructor.
+     * @param Action\Context $context
+     * @param FaqRepositoryInterface $faqRepository
+     */
+    public function __construct(Action\Context $context, FaqRepositoryInterface $faqRepository)
+    {
+        parent::__construct($context);
+        $this->faqRepository = $faqRepository;
+    }
+
+    /**
+     * @return bool
+     */
+    public function _isAllowed()
+    {
+        return $this->_authorization->isAllowed('Inchoo_ProductFAQ::productfaq');
+    }
```

Comment:
ja bi umjesto overridanja ove metode koristio konstantu. Parent metoda koristi vrijednost konstante.

```suggestion
    public const ADMIN_RESOURCE = 'Inchoo_ProductFAQ::productfaq';
```

Reply:
stavljena konstanta umjesto metode